### PR TITLE
[Communication]: Update package version for phone number after GA release

### PR DESCRIPTION
**Pull request by JianpingChen**

### CODE REVIEW

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -51,8 +51,8 @@ com.azure:azure-analytics-synapse-monitoring;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-chat;1.0.0;1.0.1
 com.azure:azure-communication-common;1.0.0;1.0.1
 com.azure:azure-communication-sms;1.0.0;1.0.1
-com.azure:azure-communication-identity;1.0.0;1.1.0-beta1
-com.azure:azure-communication-phonenumbers;1.0.0-beta.7;1.0.0
+com.azure:azure-communication-identity;1.0.0;1.1.0
+com.azure:azure-communication-phonenumbers;1.0.0-beta.7;1.0.1
```
Comment (minnieliu) [inline eng/versioning/version_client.txt:55]: shouldn't this be `com.azure:azure-communication-phonenumbers;1.0.0;1.0.1`?

File: sdk/communication/azure-communication-chat/pom.xml [hunk 2]
```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-identity</artifactId>
-      <version>1.1.0-beta1</version> <!-- {x-version-update;com.azure:azure-communication-identity;current} -->
```
Comment (minnieliu) [inline sdk/communication/azure-communication-chat/pom.xml:58]: Why are we removing beta here?

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta1 (Unreleased)
```
Comment (minnieliu) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:3]: Here as well?

File: sdk/communication/azure-communication-chat/pom.xml (hunk 2 above)
Comment (JianpingChen) [reply sdk/communication/azure-communication-chat/pom.xml:58]: I'm just restoring it to what it was before the release hack.

**Review by minnieliu — APPROVED** [review]
Looks great, thanks!

**Pull request merged**